flaskdeid/medlp.py

```diff
@@ -38,31 +28,6 @@ def annotate_phi():
     return annotate(entityTypes=["PROTECTED_HEALTH_INFORMATION"])
 
 
-@bp.route("/annotate/medication", methods=['POST'])
-def annotate_medication():
-    return annotate(entityTypes=["MEDICATION"])
-
-
-@bp.route("/annotate/condition", methods=['POST'])
-def annotate_condition():
-    return annotate(entityTypes=["MEDICAL_CONDITION"])
-
-
-@bp.route("/annotate/ttp", methods=['POST'])
-def annotate_ttp():
-    return annotate(entityTypes=["TEST_TREATMENT_PROCEDURE"])
-
-
-@bp.route("/annotate/anatomy", methods=['POST'])
-def annotate_anatomy():
-    return annotate(entityTypes=["ANATOMY"])
```

Comment:
I'd say leave these in if they're still supported in any way by hutchNER/CompMed. We're not currently using them, but they're good endpoints for other usecases to query those models

Reply:
no....should pbly change that